### PR TITLE
Fixed the .bier command

### PR DESCRIPTION
**Pull request by Davvos11**
(this is actually a fix, niels)

### CODE REVIEW

**Comment by Nierot** [comment]
https://cdn.nierot.com/memes/thonk.png

File: cogs/rolehelper.py [hunk 1]
```diff
@@ -44,6 +45,80 @@ async def _set_master(self, ctx) -> None:
...
+        """
+        Gets the last occurrence of a date (month and date)
+        :return: a datetime object on the last occurrence of the provided date
+        """
+        today = datetime.now()
+        if today.month >= month and today.day >= day:
+            return datetime(today.year, month, day)
+        else:
+            return datetime(today.year - 1, month, day)
+
```
Comment (Yuhanun) [inline cogs/rolehelper.py:121]: I'm not approving this one ;) remove the update_year stuff. peopel can just do that themselves.

Comment (Davvos11) [reply cogs/rolehelper.py:121]: The point of a bot is so people have to do less themselves smh

Comment (Davvos11) [reply cogs/rolehelper.py:121]: oh fuck i see what you mean, i forked it wrong

**Pull request closed without merging**